tutorials/getting_started/auto_overview.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: documentation
+title: Auto Overview
+---
+
+# Automatically Generated Overview Tabs
+
+MainUI will automatically generate an Overview page (id:overview). 
+This page will have four tabs:
```

Comment:
Seems, that the colon is making our website trying to parse it as yaml.
Try this:

```suggestion
This page will have four tabs.
```

Reply:
Add

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

if you do tha commit via github suggestions function to make DCO check happy directly.

Reply:
I'm working on my machine through VSCode and using the git plugin to make sure all the commits are signed off. For big stuff it's easier.

I've changed the instance you found and searched through and changed a few more `:` to `-` or `,` where appropriate to avoid the same problem elsewhere.

It looks like the preview deploys now. :-)